### PR TITLE
wider FOV DM on comma three

### PR DESCRIPTION
**Pull request by ZwX1616**

### CODE REVIEW

File: selfdrive/monitoring/driver_monitor.py [hunk 1]
```diff
@@ -27,19 +27,19 @@ def __init__(self, TICI=TICI, DT_DMON=DT_DMON):
 
     self._FACE_THRESHOLD = 0.5
     self._PARTIAL_FACE_THRESHOLD = 0.765 if TICI else 0.455
-    self._EYE_THRESHOLD = 0.25 if TICI else 0.57
-    self._SG_THRESHOLD = 0.83
-    self._BLINK_THRESHOLD = 0.62 if TICI else 0.68
-    self._BLINK_THRESHOLD_SLACK = 0.82 if TICI else 0.88
-    self._BLINK_THRESHOLD_STRICT = 0.62 if TICI else 0.68
+    self._EYE_THRESHOLD = 0.75 if TICI else 0.55
```
Comment (HaraldSchafer) [inline selfdrive/monitoring/driver_monitor.py:30]: This seems like a huge change, any particular cause for this?

File: selfdrive/ui/qt/offroad/driverview.cc [hunk 2]
```diff
@@ -66,22 +66,26 @@ void DriverViewScene::paintEvent(QPaintEvent* event) {
...
-    float x = std::abs(face_x), y = std::abs(face_y);
-    if (x <= 0.35 && y <= 0.4) {
-      alpha = 0.8 - std::max(x, y) * 0.6 / 0.375;
+    float alpha = 0.7;
+    if (face_std > 0.08) {
+      alpha = std::max(0.7 - (face_std-0.08)*7, 0.0);
     }
     const int box_size = 0.6 * rect2.height() / 2;
-    int fbox_x = valid_rect.center().x() + (is_rhd ? face_x : -face_x) * valid_rect.width();
+    int fbox_x = valid_rect.center().x() + (is_rhd ? (face_x+0.05) : -face_x) * valid_rect.width();
```
Comment (HaraldSchafer) [inline selfdrive/ui/qt/offroad/driverview.cc:88]: give 0.05 a name

File: selfdrive/monitoring/driver_monitor.py (hunk 1 above)
Comment (ZwX1616) [reply selfdrive/monitoring/driver_monitor.py:30]: the old dataset only had like ~15 people and had a fair amount of sunglasses data that didn't get filtered out.
this is fixed in new one (which looks closer to c2 distribution)

File: selfdrive/ui/qt/offroad/driverview.cc [hunk 3]
```diff
@@ -66,22 +66,26 @@ void DriverViewScene::paintEvent(QPaintEvent* event) {
   const QRect& blackout_rect = Hardware::TICI() ? rect() : rect2;
   p.fillRect(blackout_rect.adjusted(0, 0, valid_rect.left() - blackout_rect.right(), 0), bg);
   p.fillRect(blackout_rect.adjusted(valid_rect.right() - blackout_rect.left(), 0, 0, 0), bg);
+  if (Hardware::TICI()) {
```
Comment (adeebshihadeh) [inline selfdrive/ui/qt/offroad/driverview.cc:69]: can we do this based on the video size?

Comment (ZwX1616) [reply selfdrive/ui/qt/offroad/driverview.cc:69]: will comment on the other PR

**Pull request merged**